browser/js/toggl-module.js

```diff
@@ -9,190 +9,190 @@ require('moment-duration-format');
+            if (err) {
+                console.error(err);
+            }
+            saveScreenShot();
+        });
+
+        $interval(function() {
+            async.eachSeries(apiTokens(), function(apiToken, callback) {
+                new TogglClient({apiToken: apiToken})
         .getUserData({with_related_data: true}, function(err, userData) {
-          if(err) {
-            callback(err);
-            return;
-          }
-          updateUserDataContainer(userData);
-          callback(null, userData);
-        })
-      }, function(err){
-        if(err) {
-          console.error(err);
-        }
-        saveScreenShot();
+            if (err) {
+                callback(err);
+                return;
+            }
+            updateUserDataContainer(userData);
+            callback(null, userData);
+        });
+            }, function(err) {
+                if (err) {
+                    console.error(err);
+                }
+                saveScreenShot();
+            });
+
+        }, 10 * 1000);
+
+        $interval(function() {
+            $scope.userDataContainer
+            .users
+            .filter(function(u) {return u.doing_now;})
+      .forEach(function(u) {
+          var range = new Date().getTime() - Date.parse(u.current_time_entry.start);
+          u.current_time_entry.duration_output =  moment.duration(range, 'ms').format('h [hrs], m [min], s [sec]');
       });
+        }, 1000);
+    })();
+
+    var appendUserDataContainer = function(userData) {
+        var lastTimeEntry = userData.time_entries[userData.time_entries.length - 1];
+        var currentTimeEntry = null;
+        if (lastTimeEntry.duration < 0) {
+            currentTimeEntry = lastTimeEntry;
+            var range = new Date().getTime() - Date.parse(currentTimeEntry.start);
+            currentTimeEntry.duration_output = moment.duration(range, 'ms').format('h [hrs], m [min], s [sec]');
+        }
+        userData.current_time_entry = currentTimeEntry;
+        userData.doing_now = currentTimeEntry != null;
+        $scope.$apply(function() {
+            $scope.userDataContainer.users.push(userData);
+        });
+    };
 
-    }, 10 * 1000);
+    var updateUserDataContainer = function(userData) {
+        var index = $scope.userDataContainer.users.findIndex(function(u) {
+            return u.fullname === userData.fullname;
+        });
+        if (index < 0) {
+            return;
+        }
+        var lastTimeEntry = userData.time_entries[userData.time_entries.length - 1];
+        var currentTimeEntry = null;
+        if (lastTimeEntry.duration < 0) {
+            currentTimeEntry = lastTimeEntry;
+            var range = new Date().getTime() - Date.parse(currentTimeEntry.start);
+            currentTimeEntry.duration_output = moment.duration(range, 'ms').format('h [hrs], m [min], s [sec]');
+        }
+        userData.current_time_entry = currentTimeEntry;
+        userData.doing_now = currentTimeEntry != null;
```

Comment:
Identifier 'doing_now' is not in camel case.<br>Use '!==' to compare with 'null'.